reporter/src/main/kotlin/reporters/AmazonOssAttributionBuilderReporter.kt

```diff
@@ -84,8 +86,9 @@ class AmazonOssAttributionBuilderReporter : Reporter {
                 metadata
             )
 
-            val newProjectResponse = service.createNewProject(newProject, credentials).execute()
-            val newProjectBody = getBodyOrHandleError(newProjectResponse, "Unable to create a new project")
+            val newProjectBody = callService("Unable to create a new project") {
```

Comment:
From just looking at this call site the function signature of `callService()` is a bit odd, IMO. The error message (which is only used in exceptional error cases) is prominently provided as the first argument, which does not fit well the purpose of the function to "call a service"; I'd expect the service to come first, and make the error message an (optional) last argument. I believe this would be fine syntax-wise here, as we're not really needing the functionality to pass an arbitrary lambda; a reference to an existing function is enough.

Reply:
I have changed the order of parameters and also made _errorTitle_ a lambda, so that it is evaluated only in case of an error (most titles are generated using string concatenations). I did not make it optional though as all call sites pass in a value.